plugin-server/src/worker/vm/extensions/posthog.ts

```diff
@@ -52,7 +54,12 @@ export function createPosthog(hub: Hub, pluginConfig: PluginConfig): DummyPostHo
 
     return {
         capture: async (event, properties = {}) => {
-            const { timestamp = DateTime.utc().toISO(), distinct_id = distinctId, ...otherProperties } = properties
+            const {
+                timestamp = DateTime.utc().toISO(),
+                distinct_id = distinctId,
+                groups = null,
```

Comment:
What library are we emulating here? Can we make sure the interface is identical?